### ISSUE TITLE
Investigate relevancy of flag FULLSTRIP

### ISSUE THREAD

**Issue by dimztimz** [nuspell/nuspell#113]
Is this flag relevant. Seems to me like treating this like it's always TRUE always works, and even removes the need for special checks in code. Is there a case where it must be false?

**Comment by ssvb** [comment]
> Is there a case where it must be false?

Yes, it's possible to create a dictionary, which would be interpreted incorrectly with a wrong FULLSTRIP configuration. Of course create it not intentionally, but using an automated tool. Are you interested in a reduced testcase?
